src/promptflow-devkit/promptflow/_sdk/data/docker/README.md

```diff
@@ -10,5 +10,25 @@ Exported Dockerfile & its dependencies are located in the same folder. The struc
 - settings.json: a json file to store the settings of the docker image
 - README.md: the readme file to describe how to use the dockerfile
 
+
+Build Docker image:
```

Comment:
please add test to show the build result

Reply:
refer to 'test_flow_build_as_docker'

Reply:
Test has already been added: test_flex_flow_build